pkgs/misc/autotiling/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildPythonApplication, fetchPypi, i3ipc, importlib-metadata }:
+
+buildPythonApplication rec {
+  pname = "autotiling";
+  version = "1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0hwvy9bxwv9fakqqiyrkmpckxgm0z85c240p84ibdhja9sm086v0";
+  };
+
+  propagatedBuildInputs = [ i3ipc importlib-metadata ];
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://github.com/nwg-piotr/autotiling";
+    description = "Script for sway and i3 to automatically switch the horizontal / vertical window split orientation";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```
See https://github.com/nwg-piotr/autotiling/blob/126c07da532fd58df0af86f9f1741e29d6bf2db9/setup.cfg#L13

---

okay, i will do it in the same commit as the above request if i need to change something in the expression

---

done